octoprint_slic3r/profile.py

```diff
@@ -239,11 +261,18 @@ def to_slic3r_ini(cls, profile, path, display_name=None, description=None):
 	@classmethod
 	def convert_value(cls, key, value, default, sep=","):
 		try:
-			if key in float_or_percentage:
+			'''Some defaults can be instances of int whereas its values are instances of float. 
```

Comment:
Uhm.. Maybe I am not understanding correctly as well. The thing is that for some of the settings, according to the original code could be float or percentage. Running some tests, I could see the following happening:

 -Looking in Slic3r, it was said for some of the settings '(leave 0 for default)' (found in Print Settings -> Advanced)

-If any of the parameters in that list was an integer, an error happens when importing the profile

Knowing this, I did some changes so in case of integer it would also work without errors and would import it as an integer (maybe for perl or any other language used by Slic3r 0.0 would be the same as 0).